test/fixtures/output/kafka/10.golden

```diff
@@ -1,4 +1,4 @@
-  Cloud ID |     Cloud Name      |   Region ID    |       Region Name        
------------+---------------------+----------------+--------------------------
-  aws      | Amazon Web Services | ap-northeast-1 | ap-northeast-1 (Tokyo)   
-  aws      | Amazon Web Services | us-east-1      | us-east-1 (N. Virginia)  
+           Name           | Cloud |     Region      
+--------------------------+-------+-----------------
+  ap-northeast-1 (Tokyo)  | AWS   | ap-northeast-1  
```

Comment:
Previously, "AWS" used to be lowercase. Let's change that back so it matches the backend output.

Reply:
does this mean changing it to lower case for all test cases and also in the code?

Reply:
It looks like we're doing `strings.ToUpper(region.CloudId)`, so let's remove the function and update tests accordingly!